Not usable if installed with pipx

**Describe the bug**
Python now strongly discourages package installation with `pip`, even in `--user` mode instead of system-wide, because it can cause compatibility issues with system apps. The general recommendation is to use `pipx` to automatically manage a virtual environment for python apps.

**To Reproduce**
Steps to reproduce the behaviour:
1. Install Superpaper with `pipx`, e.g.:
    ```
    roy@orbox:~$ pipx install superpaper
      installed package superpaper 2.2.1, installed using Python 3.11.2
      These apps are now globally available
        - superpaper
    done! ✨ 🌟 ✨
3. run `superpaper`

**Expected behavior**
Program to launch as normal, but I don't even know if it should work with pipx.

**Actual behaviour**
```
/home/roy/.local/pipx/venvs/superpaper/lib/python3.11/site-packages
/home/roy/.config/superpaper
/home/roy/.config/superpaper/profiles
Traceback (most recent call last):
  File "/home/roy/.local/bin/superpaper", line 5, in <module>
    from superpaper.__main__ import main
  File "/home/roy/.local/pipx/venvs/superpaper/lib/python3.11/site-packages/superpaper/__main__.py", line 13, in <module>
    from superpaper.cli import cli_logic
  File "/home/roy/.local/pipx/venvs/superpaper/lib/python3.11/site-packages/superpaper/cli.py", line 9, in <module>
    from superpaper.data import CLIProfileData
  File "/home/roy/.local/pipx/venvs/superpaper/lib/python3.11/site-packages/superpaper/data.py", line 16, in <module>
    from superpaper.message_dialog import show_message_dialog
  File "/home/roy/.local/pipx/venvs/superpaper/lib/python3.11/site-packages/superpaper/message_dialog.py", line 3, in <module>
    import wx
ModuleNotFoundError: No module named 'wx'
```

**Desktop (please complete the following information):**
 - OS: Kubuntu 23.04 Lunar Lobster
 - Desktop Environment on Linux: KDE
 - Superpaper package & version: 2.2.1

Encountered the same issue, trying to install `wxPython` in the pipx venv results in fail:

Both `pipx runpip superpaper install wxPython` and `pipx inject superpaper wxPython` result in the same error.

```bash
Collecting wxPython
  Using cached wxPython-4.2.1.tar.gz (73.7 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: pillow in ./.local/share/pipx/venvs/superpaper/lib/python3.11/site-packages (from wxPython) (10.2.0)
Collecting six (from wxPython)
  Using cached six-1.16.0-py2.py3-none-any.whl.metadata (1.8 kB)
Requirement already satisfied: numpy in ./.local/share/pipx/venvs/superpaper/lib/python3.11/site-packages (from wxPython) (1.26.4)
Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Building wheels for collected packages: wxPython
  Building wheel for wxPython (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for wxPython (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [132 lines of output]
      running bdist_wheel
      running build
      /tmp/pip-install-_umwnt4a/wxpython_779207addd2d4b0fa200d35e7dee143e/build.py:43: DeprecationWarning: dep_util is Deprecated. Use functions from setuptools instead.
        from distutils.dep_util import newer, newer_group
      Will build using: "/home/user/.local/share/pipx/venvs/superpaper/bin/python"
      3.11.8 (main, Feb  7 2024, 21:52:08) [GCC 13.2.0]
      Python's architecture is 64bit
      cfg.VERSION: 4.2.1
      
      Running command: build
      Running command: build_wx
      wxWidgets build options: ['--wxpython', '--unicode', '--gtk3']
      Configure options: ['--enable-unicode', '--with-gtk=3', '--enable-sound', '--enable-graphics_ctx', '--enable-display', '--enable-geometry', '--enable-debug_flag', '--enable-optimise', '--disable-debugreport', '--enable-uiactionsim', '--enable-autoidman', '--with-sdl']
      /tmp/pip-install-_umwnt4a/wxpython_779207addd2d4b0fa200d35e7dee143e/ext/wxWidgets/configure --enable-unicode --with-gtk=3 --enable-sound --enable-graphics_ctx --enable-display --enable-geometry --enable-debug_flag --enable-optimise --disable-debugreport --enable-uiactionsim --enable-autoidman --with-sdl
      checking build system type... x86_64-pc-linux-gnu
      checking host system type... x86_64-pc-linux-gnu
      checking for toolkit... gtk
      checking for gcc... gcc
      checking whether the C compiler works... yes
      checking for C compiler default output file name... a.out
      checking for suffix of executables...
      checking whether we are cross compiling... no
      checking for suffix of object files... o
      checking whether we are using the GNU C compiler... yes
      checking whether gcc accepts -g... yes
      checking for gcc option to accept ISO C89... none needed
      checking whether we are using the Intel C compiler... no
      checking how to run the C preprocessor... gcc -E
      checking for grep that handles long lines and -e... /usr/bin/grep
      checking for egrep... /usr/bin/grep -E
      checking whether gcc needs -traditional... no
      checking for g++... g++
      checking whether we are using the GNU C++ compiler... yes
      checking whether g++ accepts -g... yes
      checking whether we are using the Intel C++ compiler... no
      checking whether we are using the Sun C++ compiler... no
      checking for ar... ar
      checking for ANSI C header files... yes
      checking for sys/types.h... yes
      checking for sys/stat.h... yes
      checking for stdlib.h... yes
      checking for string.h... yes
      checking for memory.h... yes
      checking for strings.h... yes
      checking for inttypes.h... yes
      checking for stdint.h... yes
      checking for unistd.h... yes
      checking for langinfo.h... yes
      checking for wchar.h... yes
      checking for sys/select.h... yes
      checking for cxxabi.h... yes
      checking for an ANSI C-conforming const... yes
      checking for inline... inline
      checking size of short... 2
      checking size of void *... 8
      checking size of int... 4
      checking size of long... 8
      checking size of size_t... 8
      checking size of long long... 8
      checking size of wchar_t... 4
      checking for va_copy... yes
      checking for _FILE_OFFSET_BITS value needed for large files... 64
      checking if large file support is available... yes
      checking for _LARGEFILE_SOURCE value needed for large files... no
      checking whether byte ordering is bigendian... no
      checking for std::wstring in <string>... yes
      checking for std::istream... yes
      checking for std::ostream... yes
      checking for type_traits... yes
      checking for __sync_xxx_and_fetch builtins... yes
      checking for libraries directories... /usr/lib/x86_64-linux-gnu /usr/lib
      checking for cos... no
      checking for floor... no
      checking if floating point functions link without -lm... no
      checking for sin... yes
      checking for ceil... yes
      checking if floating point functions link with -lm... yes
      checking for strtoull... yes
      checking for pkg-config... no
      checking for LIBPCRE... checking whether pcre submodule exists... yes
      checking for zlib.h >= 1.1.4... yes
      checking for zlib.h... (cached) yes
      checking for deflate in -lz... yes
      checking for png.h > 0.90... no
      checking for png.h... (cached) no
      configure: WARNING: system png library not found or too old, will use built-in instead
      checking whether png.c file exists... yes
      checking for jpeglib.h... no
      configure: WARNING: system jpeg library not found, will use built-in instead
      checking whether jpeglib.h file exists... yes
      checking for jbg_dec_init in -ljbig... no
      checking for LIBTIFF... not found via pkg-config
      checking for tiffio.h... no
      configure: WARNING: system tiff library not found, will use built-in instead
      checking whether tiff.h file exists... yes
      checking for expat.h... yes
      checking if expat.h is valid C++ header... yes
      checking for XML_ParserCreate in -lexpat... yes
      checking for LIBCURL... not found
      checking for GTK+ version...
      checking for pkg-config... no
      checking for GTK+ - version >= 3.0.0... no
      *** A new enough version of pkg-config was not found.
      *** See http://pkgconfig.sourceforge.net
      configure: error:
      The development files for GTK+ were not found. For GTK+ 2, please
      ensure that pkg-config is in the path and that gtk+-2.0.pc is
      installed. For GTK+ 1.2 please check that gtk-config is in the path,
      and that the version is 1.2.3 or above. Also check that the
      libraries returned by 'pkg-config gtk+-2.0 --libs' or 'gtk-config
      --libs' are in the LD_LIBRARY_PATH or equivalent.
      
      Error running configure
      ERROR: failed building wxWidgets
      Traceback (most recent call last):
        File "/tmp/pip-install-_umwnt4a/wxpython_779207addd2d4b0fa200d35e7dee143e/build.py", line 1563, in cmd_build_wx
          wxbuild.main(wxDir(), build_options)
        File "/tmp/pip-install-_umwnt4a/wxpython_779207addd2d4b0fa200d35e7dee143e/buildtools/build_wxwidgets.py", line 379, in main
          exitIfError(wxBuilder.configure(dir=wxRootDir, options=configure_opts),
        File "/tmp/pip-install-_umwnt4a/wxpython_779207addd2d4b0fa200d35e7dee143e/buildtools/build_wxwidgets.py", line 72, in exitIfError
          raise builder.BuildError(msg)
      buildtools.builder.BuildError: Error running configure
      Finished command: build_wx (0m2.488s)
      Finished command: build (0m2.488s)
      WARNING: Building this way assumes that all generated files have been
      generated already.  If that is not the case then use build.py directly
      to generate the source and perform the build stage.  You can use
      --skip-build with the bdist_* or install commands to avoid this
      message and the wxWidgets and Phoenix build steps in the future.
      
      "/home/user/.local/share/pipx/venvs/superpaper/bin/python" -u build.py build
      Command '"/home/user/.local/share/pipx/venvs/superpaper/bin/python" -u build.py build' failed with exit code 1.
      [end of output]
```

So far no luck

Using Debian Testing [Debian 6.6.15-2 (2024-02-04) x86_64]
Python 3.11.8

--- a/docs/installation-linux.md
+++ b/docs/installation-linux.md
@@ -12,25 +12,29 @@ This will allow Superpaper to integrate the best with your system theme and icon
 See the screenshot in the Readme taken on Manjaro KDE.
 
 
-### Step 1: Install wxPython 4.X:
+### Step 1: Install pipx and wxPython 4.X:
 
-Because of the differences between Linux distributions the installation options differ for wxPython:
+Because of the differences between Linux distributions, the installation options differ:
 
-- Arch / Manjaro: `sudo pacman -S python-wxpython`
-- Debian / Ubuntu and relatives: `sudo apt install python3-wxgtk4.0`
-- Fedora : `sudo dnf install python3-wxpython4`
+- Arch / Manjaro: `sudo pacman -S python-pipx python-wxpython`
+- Debian / Ubuntu and relatives: `sudo apt install pipx wxpython-tools`
+- Fedora : `sudo dnf install pipx python3-wxpython4`
 - Older distros with no wxPython4 package: [wxpython.org](https://wxpython.org/pages/downloads/)
   - Install the wheel if available for your OS: CentOS, Debian, Fedora and Ubuntu
+
+Using `pipx` simplifies management of the virtual environment in which Superpaper is installed, but is not a hard requirement.
+You may use a manually created venv.
 
 
 ### Step 2: Install superpaper from PyPI
 
-Superpaper is available from [PyPI](https://pypi.org/project/superpaper), and needs `Python 3.6+`. To install, run in a terminal:
+Superpaper is available from [PyPI](https://pypi.org/project/superpaper), and needs `Python 3.6+`.
+To install, run in a terminal:
 
 ```sh
-python3 -m pip install --user --upgrade superpaper
+pipx install --system-site-packages superpaper
 ```
-On some Linux setups you might need to restart or logout and login to get the menu/launcher entry to show up.
+On some Linux setups, you might need to log out and in, or restart to get the menu/launcher entry to show up.
 
 
 ## Snaps

